src/main/webapp/WEB-INF/js/collector.js

```diff
@@ -0,0 +1,6 @@
+
+
+
+$(function() { // $(document).ready(function() {
```

Comment:
iba, dovod preco je tu ten komentar?

Reply:
removed.
Na lavo je skratena syntax, komentar zobrazuje plny tvar. Zvyknem si to ta nechat lebo ked pridem vzdy po par mesiacoch spat ku jquety pozeram na to vyorane ze what. Javascript je uz taky...